### PR TITLE
add test

### PR DESCRIPTION
**Pull request by ghfvs-tester**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ## files generated by popular Visual Studio adddsfsdf-ons.
 
 # User-specific files
-*.suos
+*.suo
```
Comment (meaghanlewis) [inline .gitignore]: some comment

**Review by meaghanlewis — APPROVED** [review]

**Review by meaghanlewis-tester — COMMENTED** [review]
complete this test method

File: UnitTestProject3/UnitTestProject3/tests/UnitTest1.cs [hunk 2]
```diff
@@ -18,5 +18,9 @@ public void TestMethod2()
         public void TestMethod2()
         {
         }
+        [TestMethod]
+        public void TestMethod3()
```
Comment (meaghanlewis-tester) [inline UnitTestProject3/UnitTestProject3/tests/UnitTest1.cs:22]: complete this test method

Comment (meaghanlewis) [inline UnitTestProject3/UnitTestProject3/tests/UnitTest1.cs]: another conversation

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -11,23 +11,10 @@
...
 [assembly: AssemblyCulture("")]
 
-// Setting ComVisible to false makes the types in this assembly not visible
-// to COM components.  If you need to access a type in this assembly from
-// COM, set the ComVisible attribute to true on that type.
 [assembly: ComVisible(false)]
 
-// The following GUID is for the ID of the typelib if this project is exposed to COM
-[assembly: Guid("21cd81a4-bae6-4531-95d2-a20902b861d7")]
+[assembly: Guid("d71d6bdb-ea76-4fa3-ad31-ef3f9cee402d")]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs:16]: a new review comment

**Review by meaghanlewis — APPROVED** [review]

File: UnitTestProject1/UnitTestProject1.sln [hunk 4]
```diff
@@ -11,12 +11,15 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{21CD81A4-BAE6-4531-95D2-A20902B861D7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{21CD81A4-BAE6-4531-95D2-A20902B861D7}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{21CD81A4-BAE6-4531-95D2-A20902B861D7}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{21CD81A4-BAE6-4531-95D2-A20902B861D7}.Release|Any CPU.Build.0 = Release|Any CPU
+		{D71D6BDB-EA76-4FA3-AD31-EF3F9CEE402D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1.sln:14]: lalala

**Review by meaghanlewis — DISMISSED** [review]

File: UnitTestProject1/UnitTestProject1.sln [hunk 5]
```diff
@@ -11,12 +11,15 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{21CD81A4-BAE6-4531-95D2-A20902B861D7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1.sln:14]: single comment

**Review by meaghanlewis — DISMISSED** [review]
lalalala

**Review by meaghanlewis — APPROVED** [review]

**Review by meaghanlewis — APPROVED** [review]

File: UnitTestProject3/UnitTestProject3/tests/UnitTest1.cs [hunk 6]
```diff
@@ -18,5 +18,9 @@ public void TestMethod2()
...
+        {
+        }
     }
 }
```
Comment (meaghanlewis) [inline UnitTestProject3/UnitTestProject3/tests/UnitTest1.cs:26]: last line

**Review by meaghanlewis — COMMENTED** [review]
last line

**Pull request merged**